### ISSUE TITLE
GLES2 error on old computer on project with verbose stdout enabled

### ISSUE THREAD

**Issue by MightyPrinny** [godotengine/godot#34056]
**Godot version:**
3.2 beta 2

**OS/device including version:**
MXLinux 64-bits
GPU Intel Ironlake Mobile

**Issue description:**
This is a hardware specific issue that doesn't seem to have much impact, I tried Godot on an old laptop that doesn't support GLES3 and I get this error printed 30 times every time I run a project(any project, even a default project with nothing on the screen):

E 0:00:00:0151   _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 7	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
  <C++ Source>   drivers/gles2/rasterizer_gles2.cpp:133 @ _gl_debug_print()

This also happens in Godot 3.1.2
I can only reproduce this on this old laptop, my main laptop doesn't print anything and as far as I can tell this doesn't have any impact in games.

**Steps to reproduce:**
Try to run a project with verbose stdout enabled on a computer that's old enough and doesn't support GLES3.

**Comment by akien-mga** [comment]
This is possibly fixed already by #33844, please test with Godot 3.2 beta 3 when it's released, or a nightly build from https://hugo.pro/projects/godot-builds/

**Comment by MightyPrinny** [comment]
I get the same error on a nightly build.

**Comment by clayjohn** [comment]
Do you get this issue in 3.1.1?

**Comment by MightyPrinny** [comment]
yes, it prints multiple times, it might be the same amount(the console says there were too many errors).

**Comment by MightyPrinny** [comment]
![image](https://user-images.githubusercontent.com/18746022/70296055-f257ea00-17c7-11ea-9b46-ae1801e0ee2e.png)
Yes, multiple times.

**Comment by clayjohn** [comment]
And does it happen in 3.1.1?

**Comment by MightyPrinny** [comment]
yes the screen shot is from 3.1.1

**Comment by clayjohn** [comment]
Not gonna work on this for 3.2. Seems to only be reproducible on very old hardware and doesn't seem to break anything. Will keep it in mind for the 4.0 rewrite.

**Comment by KoBeWi** [comment]
Can anyone still reproduce this bug in Godot 3.2.3 or any later release?

**Comment by MightyPrinny** [comment]
![image](https://user-images.githubusercontent.com/18746022/100558759-6c616880-328e-11eb-8761-079804a063e6.png)
Yes, it's still happening.
